src/content/reference/cli.mdx

```diff
@@ -77,9 +77,40 @@ The following flags can be used (very similar to `docker build`):
 | _--secret_       |  (list)  | Secret files exposed to the build. Format: `id=mysecret,src=/local/secret` |
 | _--export-cache_ | (string) | Image tag for exported cache when build (optional)                         |
 | _--cache-from_   |  (list)  | List of cache source images (optional)                                     |
+| _--platform_     | (string) | Specify which platform to build the container image for (optional)                                     |
+
 
 >  You can also use the `-f` to point to a Dockerfile. In this mode, `okteto build` will ignore your Okteto manifest, and directly build the image defined in the Dockerfile. Use this to build images that are not defined on your [Okteto manifest](https://www.okteto.com/docs/reference/manifest/).
 
+With `oktet build --platform` you can specify the platform (or architecture) for which you'd like to build the container images for. Suppose you want to deploy your web application on a Kubernetes cluster that consists of nodes running on both x86-64 and ARMv7 architectures. By using the multiplatform images
```

Comment:
```suggestion
With `oktet build --platform` you can specify the platform (or architecture) on which you'd like to build the container images. For example, you could use a multiplatform image and the `okteto build --platform` command to deploy your web application on a Kubernetes cluster that consists of nodes running on both x86-64 and ARMv7 architectures.
```

@nitishfy Thanks so much for this PR 🙇 I've made this non-blocking suggestion to be a bit more concise and direct (imo), but happy for you to object or propose alternative wording 🙂